yaksh/models.py

```diff
@@ -68,43 +68,6 @@
                 ('completed', 'Completed'),
               )
 
-instructions_data = dedent("""\
```

Comment:
After re-reading your comment, I understand that the removal of instructions_data would have caused a problem in creating a demo quiz. Hence, i have removed the instructions argument from create_demo_quiz model method. Instead while creating the demo course, it will take the default value as defined in the instruction attribute of quiz model method.